### ISSUE TITLE
Update focus indicator on open Accordions

### ISSUE THREAD

**Issue by syarwick** [loyolauniversitychicago/loyola-template-4.0#53]
When the accordions and nested accordions are open and the accordion headers are maroon, it is hard to see the black focus indicator.

This is not a WCAG failure but may confuse users.

**Comment by stephenravenscraft** [comment]
The focus state is updated. I've added CSS in the repository accordion. This will need to be re-checked.
- - - - -
:focus-within
    outline: .25rem dashed darken($patina, 5%);
    outline-offset: .25rem;
- - - - -
Contrast Checked: Contrast Ratio 4.84:1
